nats/src/main/proto/ua/lviv/iot/parking_spot.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package ua.lviv.iot;
+
+enum ParkingSpotSize {
+  COMPACT = 0;
+  LARGE = 1;
+  MOTORBIKE = 2;
+}
```

Comment:
how the zero element of an enum is special and documentation recommends to use it?

Reply:
I see, zero element is a default value. Thanks)